rest_framework_simplejwt/key_id.py

```diff
@@ -0,0 +1,33 @@
+import base64
+import hashlib
+from cryptography.hazmat.primitives import serialization
+from jwt.algorithms import RSAAlgorithm
+
+
+class BaseKeyID:
+    def __init__(self, public_key):
+        prepared_key = RSAAlgorithm.prepare_key(None, public_key)
+        self.public_bytes = prepared_key.public_bytes(encoding=serialization.Encoding.DER,
+                                                      format=serialization.PublicFormat.SubjectPublicKeyInfo)
+
+    def __str__(self):
+        raise NotImplemented
```

Comment:
```suggestion
        raise NotImplementedError
```
What flake8 rose. I think it's best with this than `NotImplemented` due to their different use cases: https://docs.python.org/3.6/library/exceptions.html#NotImplementedError